conda_lock/models/channel.py

```diff
@@ -1,126 +1,14 @@
-"""
```

Comment:
Most of this has been extracted to `conda_lock/models/package_source.py`, which contains the shared logic common to both channels and pip repositories.